src/components/dialogs/parameters/shortcircuit/short-circuit-parameters.tsx

```diff
@@ -0,0 +1,189 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import React, {
+    FunctionComponent,
+    useCallback,
+    useEffect,
+    useState,
+} from 'react';
+import { Grid } from '@mui/material';
+import {
+    SHORT_CIRCUIT_INITIAL_VOLTAGE_PROFILE_MODE,
+    SHORT_CIRCUIT_PREDEFINED_PARAMS,
+    SHORT_CIRCUIT_WITH_FEEDER_RESULT,
+    SHORT_CIRCUIT_WITH_LOADS,
+    SHORT_CIRCUIT_WITH_NEUTRAL_POSITION,
+    SHORT_CIRCUIT_WITH_SHUNT_COMPENSATORS,
+    SHORT_CIRCUIT_WITH_VSC_CONVERTER_STATIONS,
+} from '../../../utils/field-constants';
+import {
+    CheckboxInput,
+    FieldLabel,
+    RadioInput,
+    SelectInput,
+    SwitchInput,
+} from '@gridsuite/commons-ui';
+
+import { INITIAL_TENSION, STATUS } from '../../../utils/constants';
+import { gridItem, GridSection } from '../../dialogUtils';
+import { useIntl } from 'react-intl';
+import { green, red } from '@mui/material/colors';
+import { useWatch } from 'react-hook-form';
+import TensionTable from './short-circuit-tension-table';
+import {
+    useIntlInitialVoltageProfileMode,
+    useIntlPredefinedParametersOptions,
+    useStatus,
+} from './short-circuit-parameters-utils';
+
+interface ShortCircuitFieldsProps {
+    resetAll: (predefinedParams: INITIAL_TENSION) => void;
+}
+
+const ShortCircuitFields: FunctionComponent<ShortCircuitFieldsProps> = ({
+    resetAll,
+}) => {
+    const intl = useIntl();
+
+    const [status, setStatus] = useState(STATUS.SUCCESS);
+    const [isChanged, setIsChanged] = useState(false);
+
+    const watchInitialVoltageProfileMode = useWatch({
+        name: SHORT_CIRCUIT_INITIAL_VOLTAGE_PROFILE_MODE,
+    });
+    const watchPredefinedParams = useWatch({
+        name: SHORT_CIRCUIT_PREDEFINED_PARAMS,
+    });
+
+    const styles = {
+        cell: {
+            display: 'flex',
+            alignItems: 'center',
+            textAlign: 'center',
+            boxSizing: 'border-box',
+            flex: 1,
+            cursor: 'initial',
+        },
+        succeed: {
```

Comment:
Do you think it is better to add success colors  and fail color on the global theme on app-wrapper?

( adding status key to lightTheme and darkTheme with success and fail as properties )